branchingQuestion.js

```diff
@@ -43,12 +43,66 @@ H5P.BranchingQuestion = (function () {
       var icon = document.createElement('img');
       icon.classList.add('h5p-branching-question-icon');
       icon.src = self.getLibraryFilePath('branching-question-icon.svg');
+      const close = createCloseElement();
 
       wrapper.appendChild(icon);
+      wrapper.appendChild(close);
 
       return wrapper;
     };
 
+    /**
+     * Create close element and register events
+     *
+     * @return {Element} close
+     */
+    const createCloseElement = function () {
+      const close = document.createElement('div');
+      close.classList.add('h5p-branching-question-close');
+      close.tabIndex = 0;
+      close.setAttribute('aria-label', H5P.t('close'));
+      close.setAttribute('type', 'button');
+      close.setAttribute('title', H5P.t('close'));
+
+      close.addEventListener('keyup', function (event) {
+        // Add support for space and enter
+        if (event.code === 'Enter' || event.code === 'Space') {
+          closeDialog();
+        }
+      });
+
+      close.onclick = function (e) {
+        // Add clickevent
+        closeDialog();
+      };
+
+      return close;
+    };
+
+    /**
+     * Remove BQ and navigate back to its previous position
+     */
+    const closeDialog = function () {
+      const overlay = self.parent.libraryScreen.overlay;
+      if (overlay) {
+        // TODO: When does this code every run?!
```

Comment:
This TODO should be resolved. I.e: I don't understand why this code was added if you don't know when it is executed?